build.sbt

```diff
@@ -205,6 +205,16 @@ lazy val http4s0170 =
 lazy val http4s0170_2_11 = http4s0170(scala211).dependsOn(shared_2_11)
 lazy val http4s0170_2_12 = http4s0170(scala212).dependsOn(shared_2_12)
 
+
+lazy val http4s0180 =
+  (project in file("scalapact-http4s-0-18-0"))
```

Comment:
Nothing you need to do but I'm planning to drop the minor versions from the folder names as they're unhelpful.